deepchem/molnet/load_function/uspto_datasets.py

```diff
@@ -4,93 +4,117 @@
 This file contains loaders for synthetic reaction datasets from the US Patenent Office. http://nextmovesoftware.com/blog/2014/02/27/unleashing-over-a-million-reactions-into-the-wild/.
```

Comment:
"Patent"

Reply:
Whoops! this was in the original file as well, I shall fix it.